src/utils.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union
+from io import BytesIO
+from PIL import Image
+
+import numpy as np
+
+from viam.logging import getLogger
+from viam.media.video import CameraMimeType
+from viam.media.video import ViamImage
+
+LOGGER = getLogger(__name__)
+
+SUPPORTED_IMAGE_TYPE = [
+    CameraMimeType.JPEG,
+    CameraMimeType.PNG,
+    CameraMimeType.VIAM_RGBA,
+]
+
+LIBRARY_SUPPORTED_FORMATS = ["JPEG", "PNG", "VIAM_RGBA"]
+
+def create_empty_rgb_image(height: int, width: int) -> np.ndarray:
+    """
+    Create an empty RGB image with the specified height and width
+    Args:
+        height (int): Height of the image
+        width (int): Width of the image
+    Returns:
+        np.ndarray: An empty RGB image of the specified size
+    """
+    return np.zeros((height, width, 3), dtype=np.uint8)
+
+def decode_image(image: Union[Image.Image, ViamImage, np.ndarray]) -> np.ndarray:
+    """
+    Decode image to BGR numpy array.
+    Args:
+        raw_image (Union[Image.Image, RawImage])
```

Comment:
Comment doesn't match type annotation in the function. I'm not convinced we need this docstring at all: I don't think it adds anything useful.